app/assets/stylesheets/govuk_publishing_components/components/_summary-list.scss

```diff
@@ -14,12 +13,27 @@
   &:nth-last-of-type(1) {
     border-bottom: 0;
   }
+
+  .govuk-summary-list {
+    clear: both;
+  }
 }
 
-.gem-c-summary-list__edit-section-link {
-  position: absolute;
-  top: 0;
-  right: 0;
+.gem-c-summary-list__group-title {
+  @include govuk-media-query($from: tablet) {
+    display: inline-block;
+    margin-right: govuk-spacing(3);
+  }
+}
+
+.gem-c-summary-list__group-actions-list {
+  margin-bottom: govuk-spacing(3);
+
+  @include govuk-media-query($from: tablet) {
+    display: inline-block;
```

Comment:
`inline-block` isn't needed if this element is floated.

Reply:
Good spot, a leftover from the first iteration (where on mobile they were kept inline, but that would've made actions at the group level inconsistent with actions at the item level).